handbook/engineering/campaigns/goals.md

```diff
@@ -13,6 +13,32 @@ What will get us there?
 - **Better debugging experience:** It can be challenging to debug a more complex campaign. Figuring out what went wrong and how to fix it is a slow iterative loop.
 - **User validation:** The above issues were revealed by user testing. Once we believe we have fixed them, we need to validate this with further user testing.
 
+
```

Comment:
The above is more of a milestone than a goal, and this milestone belongs to the "Grow Adoption" goal. (I will make that change later today.)